### PR TITLE
[MM-29802] Add end-2-end tests to generate token for user

### PR DESCRIPTION
**Pull request by lucianomagrao**
#### Summary

Add e2e tests to `mmctl token generate`command.

#### Ticket Link

Fixes: https://github.com/mattermost/mattermost-server/issues/15998
JIRA: https://mattermost.atlassian.net/browse/MM-29802

### CODE REVIEW

**Review by jespino — APPROVED** [review]
Looks good to me. Thanks @lucianomagrao! :tada:

File: commands/token_e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+)
+
+func (s *MmctlE2ETestSuite) TestTokenGenerateForUserCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	tokenDescription := model.NewRandomString(10)
+
+	previousVal := s.th.App.Config().ServiceSettings.EnableUserAccessTokens
+	s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableUserAccessTokens = true })
+	defer s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableUserAccessTokens = *previousVal })
```
Comment (mgdelacroix) [inline commands/token_e2e_test.go:23]: nit: there is no need to save and restore the value of the config as it is initialised on each `s.SetupTestHelper().InitBasic()`

**Review by mgdelacroix — APPROVED** [review]
LGTM, thanks a lot @lucianomagrao!! :confetti_ball:

**Review by lindy65 — APPROVED** [review]
Thanks @lucianomagrao @mgdelacroix - looks good to me 👍

**Comment by hanzei** [comment]
Thanks for the contribution @lucianomagrao :rocket:

**Pull request merged**